microprofile/4.1/opentracing/2.0/TCKResults.adoc

```diff
@@ -31,83 +31,83 @@ Java 8 Test results:
 ----
 <?xml version="1.0" encoding="UTF-8" ?>
 <testsuites>
-  <testsuite errors="0" failures="0" hostname="ebcprs2352mcdbibf-n.fyre.ibm.com" id="1" name="ClientRegistrarTests" package="org.eclipse.microprofile.opentracing.tck" tests="4" time="0.531" timestamp="17 Mar 2021 09:21:22 GMT">
-      <testcase classname="org.eclipse.microprofile.opentracing.tck.ClientRegistrarTests" name="testClientRegistrar" time="0.135" />
-      <testcase classname="org.eclipse.microprofile.opentracing.tck.ClientRegistrarTests" name="testClientRegistrarExecutorAsync" time="0.080" />
-      <testcase classname="org.eclipse.microprofile.opentracing.tck.ClientRegistrarTests" name="testClientRegistrarExecutor" time="0.130" />
-      <testcase classname="org.eclipse.microprofile.opentracing.tck.ClientRegistrarTests" name="testClientRegistrarAsync" time="0.186" />
+  <testsuite errors="0" failures="0" hostname="ebcprs0899qmwpw6c-n.fyre.ibm.com" id="1" name="ClientRegistrarTests" package="org.eclipse.microprofile.opentracing.tck" tests="4" time="0.541" timestamp="3 Jul 2021 06:23:37 GMT">
```

Comment:
Need to update the Open Liberty versions and links to be for `21.0.0.8-beta`

Reply:
The lIne 6 and 10 need to be updated accordingly.

Reply:
I wish github would let me comment on unedited lines!

Reply:
updated